external/aws/variables.tf

```diff
@@ -1,27 +1,31 @@
-variable "instance_types" {
-  type    = "map"
-  default = {
-    "dev" = "t3.nano"
-    "prod" = "t3.large"
-  }
+variable "homebase_shape" {}
+variable "proxy_shape" {}
+variable "elk_shape" {}
+
+variable "image_username" {
+  default = "ubuntu"
 }
 
-variable "amis" {
-  type = "map"
-  default = {
-    "ubuntu18.04" = "ami-0e3e4660d8725dd31"
-    "kali 2018.3a" = "ami-0f95cde6ebe3f5ec3"
-  }
+variable "ssh_config_path" {
+  default = "~/.ssh"
 }
 
 variable "env" {
```

Comment:
this variable doesn't appear to be used

Reply:
Sorry, this is my unfamiliarity with GH. Line 17 is what I clicked, I'm not sure why it went up and not down. The `env` variable isn't used